cmake/TupleTreeGenerator.cmake

```diff
@@ -259,6 +259,8 @@ function(
       "${SCRIPTS_ROOT_DIR}/tuple-tree-generate-typescript.py" --namespace
       "${NAMESPACE}" --root-type "${ROOT_TYPE}" --output "${OUTPUT_PATH}"
       "${YAML_DEFINITIONS}"
+    COMMAND "${CMAKE_BINARY_DIR}/ts-ttg-test/prep-ts-serdes-test.sh"
```

Comment:
* `prep`? No abbreviations.
* What's serdes?

Reply:
`prep` is for preparation, it compiles the `model.ts` file to javascript
serdes stands for `ser`ialization-`des`erialization, I'll change name to make it clearer

Reply:
Renamed `serdes` to `roundtrip` to be consistent w.r.t. python